tests/bgp/test_route_aggregation.py

```diff
@@ -0,0 +1,187 @@
+    gather_info['duthost'].shell("sudo config loopback add Loopback12")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback12 1.1.2.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback12 1:1:1:2::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback13")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback13 1.1.3.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback13 1:1:1:3::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback14")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback14 1.1.4.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback14 1:1:1:4::/64")
+    gather_info['duthost'].shell("sudo config loopback add Loopback15")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback15 1.1.5.1/24")
+    gather_info['duthost'].shell("sudo config interface ip add Loopback15 1:1:1:5::/64")
+
+    cmd = 'vtysh -c "config" -c "route-map AGG_TEST_1 permit 10" -c "set as-path prepend 11111" -c "exit" -c "end"'
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    cmd = 'vtysh -c "config" -c "route-map AGG_TEST_2 permit 10" -c "set as-path prepend 22222" -c "exit" -c "end"'
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    cmd = 'vtysh{} -c "config" -c "router bgp {}" -c "address-family ipv4 unicast" \
+          -c "network 1.1.1.0/24 route-map AGG_TEST_1" -c "network 1.1.2.0/24 route-map AGG_TEST_1" \
+          -c "network 1.1.3.0/24 route-map AGG_TEST_2" -c "network 1.1.4.0/24 route-map AGG_TEST_2" \
+          -c "network 1.1.5.0/24" -c "address-family ipv6 unicast" -c "network 1:1:1:1::/64 route-map AGG_TEST_1" \
+          -c "network 1:1:1:2::/64 route-map AGG_TEST_1" -c "network 1:1:1:3::/64 route-map AGG_TEST_2" \
+          -c "network 1:1:1:4::/64 route-map AGG_TEST_2" -c "network 1:1:1:5::/64 route-map AGG_TEST_2"'.format(
+              gather_info['cli_options'], gather_info['dut_asn'])
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    verify_route_agg(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], "5", False)
+    check_baseline(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], num_v4_routes,
+                   num_v6_routes)
+
+    # Configure summary-only route aggregation
+    agg_configuration(True, gather_info['dut_asn'], gather_info['duthost'],
+                      gather_info['cli_options'], "summary-only", "summary-only")
+    logger.debug(gather_info['duthost'].shell('vtysh -c "show run bgp"', module_ignore_errors=True)['stdout'])
+    verify_route_agg(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], "1", False)
+
+    # Remove the aggregate configuration
```

Comment:
same here.

Reply:
What is the same?  This is two functions that perform the repeated tasks.